styles.css

```diff
@@ -0,0 +1,72 @@
+*{
+    margin: 0;
+    
+}
+.app {
+    padding: 50px;
+    background: #1F2937;
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+    align-items: center;
+   
+}
+.header {
+    display: flex;
+    flex-direction: column;
+    gap: 10px; 
+    
+}
+h1{
```

Comment:
-Generally, we prefer  to give class name for our elements, and style them with their class name. Imagine, that we have four different h1. And 2 of them need to have `font-weight:400` other 2 of them `font-weight:600` and different font family. Isnt it better to use class name for styling them to parse them?